README.md

```diff
@@ -588,7 +588,7 @@ Other Style Guides
     }
 
     // good
-    const foo = function bar() {
+    const foo = function foo () {
```

Comment:
yep! just like that.

If you're using recursion, it also helps disambiguate where the recursive call occurs.